src/utils/create-element.ts

```diff
@@ -0,0 +1,15 @@
+interface comment {
+  tag: string;
+  className?: string;
+  innerText?: string;
+}
+export const createNewElement = ({ tag, className, innerText }: comment) => {
+  const newElement = document.createElement(tag);
+  if (className) {
+    newElement.className = className;
+  }
+  if (innerText) {
+    newElement.innerText = innerText;
+  }
+  return newElement;
```

Comment:
if classname is optional it will be undefined if not passed.

why not just set the className as className?
if it isn't passed it will try to set undefined as the className which should be equivalent

Reply:
const newElement: HTMLElement
Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'

Reply:
same for inner Text